### ISSUE TITLE
[Bug]: ValueError: 'disable' is not in list

### ISSUE THREAD

**Issue by lestatic** [custom-components/ble_monitor#813]
### What happened?

Не подключает mijia bluetooth hygrothermograph 2, пишет: не удалось настроить. На mijia bluetooth hygrothermograph 2 стоит кастомная прошивка

Does not connect mijia bluetooth hygrothermograph 2, writes: failed to configure. Mijia bluetooth hygrothermograph 2 has custom firmware

### Relevant log output

Эта ошибка возникла в кастомной интеграции.
This error occurred in custom integration.

```shell
Logger: homeassistant.config_entries
Source: custom_components/ble_monitor/__init__.py:301
Integration: Passive BLE monitor (documentation, issues)
First occurred: 20:25:01 (1 occurrences)
Last logged: 20:25:01

Error setting up entry Bluetooth Low Energy Monitor for ble_monitor
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.8/site-packages/homeassistant/config_entries.py", line 313, in async_setup
    result = await component.async_setup_entry(hass, self)  # type: ignore
  File "/home/homeassistant/.homeassistant/custom_components/ble_monitor/__init__.py", line 301, in async_setup_entry
    list(BT_INTERFACES.values()).index(DEFAULT_BT_INTERFACE)
ValueError: 'disable' is not in list
```

**Comment by Magalex2x14** [comment]
@lestatic Добро пожаловать! Здесь в основном пользуются английским, поскольку это общепринятый язык общения в международных сообществах. Иногда тебе могут даже не ответить, если вопрос задан не на английском.

Приведенная тобой ошибка чаще всего значит, что HomeAssistant (интеграция ble_monitor) потерял  bluetooth-интерфейс. Это бывает, если BT-адаптер был выбран в настройках интеграции, но пропал из системы при очередном запуске интеграции (причины могут быть разные - плохой контакт в USB-разъеме, например). Еще это иногда бывает после обновления HomeAssistant OS - дополнительная перезагрузка системы как правило помогает тогда.

Напиши поподробнее о своей системе - как у тебя установлен HomeAssistant, какой bluetooth-адаптер, и какая версия ble_monitor. Дальше лучше продолжить на английском, если для тебя это не большая проблема.

---

@lestatic Welcome! English is mainly used here, as it is the common language of communication in international communities. Sometimes you may not even get an answer if the question is not in English.

The error you gave most often means that HomeAssistant (ble_monitor integration) lost your bluetooth interface. This happens if the BT adapter was selected in the integration settings, but disappeared from the system the next time the integration was started (reasons may be different - a bad contact in the USB connector, for example). It also sometimes happens after updating HomeAssistant OS - an additional reboot of the system usually helps then.

Write in more detail about your system - how you have HomeAssistant installed, what bluetooth adapter, and what version of ble_monitor. Then it is better to continue in English, if this is not a big problem for you.

**Comment by lestatic** [comment]
Home assistant is installed on ubuntu. I am not a very big specialist and have just started to master it. mijia bluetooth hygrothermograph 2 connected via gateway 3 with custom firmware and updated bluetooth firmware.

**Comment by Magalex2x14** [comment]
So you don't have a bluetooth interface on the HomeAssistant host? No wonder then - ble_monitor only works with bluetooth interfaces directly or with ESPHome gateways. You need another integration. You can ask for advice and discuss this issue in the Russian-speaking Telegram group https://t.me/homassistant or https://t.me/SprutAI_HomeAssistant (I know that there is a great integration (from what I heard) for Gateway 3, but I I haven't used it and I don't have relevant links).

**Comment by Ernst79** [comment]
Regardless of the issue of not having a BT adapter, this error should not have occurred, it should just use "disable" (with a warning that there is no BT adapter). 

I added the following commit https://github.com/custom-components/ble_monitor/pull/812/commits/5d5df3fd899f598d4f8c5aa02c577e2069e20477 to fix this error. Will be included in the next beta.

**Comment by Ernst79** [comment]
Fixed the error in 8.5.2.
